docs/UsersGuide/source/LAMGrids.rst

```diff
@@ -52,11 +52,14 @@ utility (UPP) is currently unable to process data on the native FV3 gnomonic gri
 model data are interpolated to a Lambert conformal grid (the write component grid) in order for UPP to
 read in and correctly process the data.
 
+.. 
+  COMMENT: Is this still the case? 
+
```

Comment:
RRFS_CONUS_13km is now larger, so I think discussion about areas of the US being close to the boundaries is no longer valid.